server/planning/planning/planning.py

```diff
@@ -1011,6 +1011,13 @@ def _create_update_assignment(
             if planning_original.get("name") != planning_updates.get("name"):
                 assignment["name"] = planning["name"] if not translated_value and translated_name else translated_name
 
+            # If the coverage assignee has been changed and workflow status is active
+            if original.get("workflow_status") == WORKFLOW_STATE.ACTIVE and self.is_coverage_assignment_modified(
```

Comment:
I'm wondering if this should be excluding states instead of explicitly checking for 1.
`original.get("workflow_status") != WorkflowStates.DRAFT` might suffice for the state checking here